### DISCUSSION TITLE
Start the bot immediately

### DISCUSSION THREAD

**Discussion by moose999** [LiveHelperChat/livehelperchat#2124] [Q&A] [answered]
Hi,

How do I get the bot to start immediately? I mean, as soon as they click the icon on the widget.

Currently, a visitor has to type something into the chat box before the bot will show my welcome message. 

I have set my welcome message trigger as "default".

Many thanks,
Justin

**Comment by remdex** [comment, marked as answer]
Watch youtube videos about various start chat options. You need to look at start chat form settings.

**Comment by moose999** [comment]
Perfect got it, many thanks!
